Makefile

```diff
@@ -47,3 +47,17 @@ push-aur:
 clean:
 	rm -f $(MANS)
 	rm -f sbctl
+
+.PHONY: deps
+deps:
+	go get honnef.co/go/tools/cmd/staticcheck@2020.1.6
+
+.PHONY: lint
+lint: deps
```

Comment:
I would prefer to be able to obtain `staticcheck` on my own; perhaps CI should do `make deps lint` for the dependency, but a user can obtain in in other ways.

Reply:
I would have to disagree here as that I think linting tools should be versioned just like dependencies. That being said, I can remove the call to `deps` when running `lint`, that way you can still use your own version.

Reply:
I don't disagree with the versioning, definitely. That sounds good to me.